org.adempiere.base/src/org/adempiere/util/ModelInterfaceGenerator.java

```diff
@@ -909,6 +909,9 @@ public static void generateSource(String type, String sourceFolder, String packa
 		{
 			pstmt = DB.prepareStatement(sql.toString(), null);
 			rs = pstmt.executeQuery();
+			if (!rs.isBeforeFirst())
+				System.out.println("No data found for the table with name " + tableName);
+

```

Comment:
See https://docs.oracle.com/en/java/javase/17/docs/api/java.sql/java/sql/ResultSet.html#isBeforeFirst()

It is an optional feature of JDBC, we shouldn't depends on this.

Reply:
Changed